evgeny.sirchenko/homework.rb

```diff
@@ -0,0 +1,3 @@
+p a = Array.new(3) { Array.new(3) {rand(-10..10)} }
+puts a = a.flatten.find_all {|ar| ar  > 0}
```

Comment:
```suggestion
puts a = a.flatten.find_all { |ar| ar  > 0 }
```

https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually